### PR TITLE
Update Prediction.cs

### PR DESCRIPTION
**Pull request by legacy3**

### CODE REVIEW

File: Prediction.cs [hunk 1]
```diff
@@ -432,7 +235,7 @@ internal static PredictionOutput GetPrediction(PredictionInput input, bool ft, b
             if (ft)
             {
                 //Increase the delay due to the latency and server tick:
-                input.Delay += Game.Ping / 2000f + 0.06f;
+                input.Delay += Game.Ping / 2000f + 0.05f;
```
Comment (ChewyMoon) [inline Prediction.cs]: broscience to the max

**Comment by nabbhacker** [comment]
:kappa:

File: Prediction.cs (hunk 1 above)
Comment (legacy3) [inline Prediction.cs]: It's actually the correct value. If you calculate it why would it be broscience?

**Comment by jQueryBase64MyAss** [comment]
Merge

**Comment by legacy3** [comment]
Not sure if i break something ¯\_(ツ)_/¯

File: Prediction.cs [hunk 2]
```diff
@@ -874,47 +608,37 @@ public static PredictionOutput GetPrediction(PredictionInput input)
...
-            /// <summary>
-            /// Gets the prediction.
-            /// </summary>
-            /// <param name="input">The input.</param>
-            /// <returns>PredictionOutput.</returns>
             public static PredictionOutput GetPrediction(PredictionInput input)
             {
                 var mainTargetPrediction = Prediction.GetPrediction(input, false, true);
-                var posibleTargets = new List<PossibleTarget>
+                var posibleTargets = new List<PosibleTarget>
```
Comment (jQueryBase64MyAss) [inline Prediction.cs]: Including typos

Comment (legacy3) [inline Prediction.cs]: sorry i dont englado

File: Prediction.cs (hunk 1 above)
Comment (ChewyMoon) [inline Prediction.cs]: I would love to see your calculations as this is a very interesting and less known topic about League of Legends.

Comment (legacy3) [inline Prediction.cs]: I would be pleased to show you the calculations. Will prepare a simple essay for you!

**Pull request closed without merging**